packages/gcp/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "2.44.1"
```

```suggestion
- version: "2.45.0"
```
enhancements bump minor, also update manifest.